frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueCard.tsx

```diff
@@ -53,15 +62,30 @@ export function MissionQueueCard({ mission, onDeleteMission }: MissionQueueCardP
     return (
         <StyledMissionCard key={mission.id} variant="default" style={{ boxShadow: tokens.elevation.raised }}>
             <HorizontalContent>
-                <HorizontalNonButtonContent onClick={routeChange}>
+                <HorizontalNonButtonContent>
```

Comment:
Can move the buttons outside of this style to avoid the need for moving route

Reply:
I am worried that this would overcomplicate the styling, since it currently uses a grid display

Reply:
![image](https://github.com/equinor/flotilla/assets/33518988/3463e340-d3b9-498e-936e-8bec05598a77)

Moving some of the "grid-template-columns" arguments between the styles as this gives the same result as the current solution

Reply:
I have tried to be more consistent with the current approach by moving the buttons to the HorizontalNonButtonContent now, so all the buttons are near each other at the end of the row.